src/Sylius/Bundle/UserBundle/UserEvents.php

```diff
@@ -13,7 +13,7 @@
 
 namespace Sylius\Bundle\UserBundle;
 
-final class UserEvents
+interface UserEvents
```

Comment:
I assume, that in this case, we should treat it as a BC Break :( /cc @pamil

Reply:
Only if someone, for an unknown reason, instantiated these classes in their code...

Reply:
I'm OK as long as we mention that in the UPGRADE file.